docs/src/main/asciidoc/logging.adoc

```diff
@@ -9,10 +9,93 @@ include::./attributes.adoc[]
 
 This guide explains logging and how to configure it.
 
+Internally, Quarkus uses JBoss Log Manager and the JBoss Logging facade. +
+You can use the JBoss Logging facade inside your code as it's already provided,
+or any of the supported Logging API listed in the next chapter as Quarkus will send them to JBoss Log Manager.
+
+All the logging configuration will then be done inside your `application.properties`.
+
+== Supported Logging APIs
+
+Applications and components may use any of the following APIs for logging, and the logs will be merged:
+
+* JDK `java.util.logging` (also called JUL)
+* https://github.com/jboss-logging/jboss-logging[JBoss Logging]
+* https://www.slf4j.org/[SLF4J]
+* https://commons.apache.org/proper/commons-logging/[Apache Commons Logging]
+
+Internally Quarkus uses JBoss Logging, you can also use it inside your application so that no other dependencies should be added for your logs.
+
+[source,java]
+----
+import org.jboss.logging.Logger;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/hello")
+public class ExampleResource {
+
+    private static final Logger LOG = Logger.getLogger(ExampleResource.class);
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String hello() {
+        LOG.info("Hello");
+        return "hello";
+    }
+}
+----
+
+NOTE: If you use JBoss Logging but one of your libraries uses a different logging API, you may need to configure a link:#logging-adapters[Logging Adapter].
+
+=== What about Apache Log4j 2 ?
+
+link:https://logging.apache.org/log4j/2.x/[Log4j 2] is a logging implementation, it contains a logging backend and a logging facade.
+Quarkus uses the JBoss Log Manager backend, so you will need to include the `log4j2-jboss-logmanager` library to route Log4j 2 logs to JBoss Log Manager.
+
+[source,xml]
+----
+    <dependency>
+      <groupId>org.jboss.logmanager</groupId>
+      <artifactId>log4j2-jboss-logmanager</artifactId>
+    </dependency>
+----
+
+You can then use the Log4J 2 API inside your application.
+
+WARNING: do not include any Log4j 2 dependencies, the `log4j2-jboss-logmanager` library includes what's needed to use Log4j 2 as a logging facade.
+
+NOTE: if you still use the legacy Log4j version 1, you need to use `log4j-jboss-logmanager` instead.
+
+== Logging levels
+
+JBoss Logging supports Apache style logs levels:
+
+- **FATAL**: A critical service failure/complete inability to service requests of any kind. *It has a higher order than **SEVERE**.*
```

Comment:
You could use descriptive list here and the list below (https://asciidoctor.org/docs/user-manual/#description-list) with `horizontal` style. THe final appearance is very similar to what it's written and more semantycally accurate.

Reply:
I didn't kow this type of list, try it and I do think it looks better, thanks for the suggestion.